src/plots/Vector/VectorAttributes.code

```diff
@@ -77,7 +77,8 @@ Definition:
 bool
 VectorAttributes::ChangesRequireRecalculation(const VectorAttributes &obj)
 {
-    return ((useStride != obj.useStride) ||
+    return ((glyphLocation != obj.glyphLocation) ||
+            (useStride != obj.useStride) ||
             (stride != obj.stride) ||
             (glyphLocation != obj.glyphLocation) ||
             (nVectors != obj.nVectors) ||
```

Comment:
You need to modify the _setattr, _getattr functions in the code file to account for any field names that have changed or been removed (they are now considered legacy).
You also need to modify the ProcessOldVersions for the same reason.

Reply:
I have added the needed code for ColorByMag -> ColorByMagnitude